ledger/participant-integration-api/src/main/scala/platform/configuration/ExtendedDropwizardExports.scala

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.configuration
+
+import java.util.Collections
+import scala.jdk.CollectionConverters._
+
+import com.codahale.metrics.{MetricRegistry, Snapshot}
+import io.prometheus.client.Collector.MetricFamilySamples
+import io.prometheus.client.dropwizard.DropwizardExportsAccess
+
+final class ExtendedDropwizardExports(metricRegistry: MetricRegistry)
+    extends DropwizardExportsAccess(metricRegistry) {
+
+  override def fromSnapshotAndCount(
+      dropwizardName: String,
+      snapshot: Snapshot,
+      count: Long,
+      factor: Double,
+      helpMessage: String,
+  ): MetricFamilySamples = {
+
+    val basicMetricFamilySamples =
+      super.fromSnapshotAndCount(dropwizardName, snapshot, count, factor, helpMessage)
+
+    val additionalMetrics = basicMetricFamilySamples.samples.asScala ++ List(
```

Comment:
This one should be named `extendedMetrics`, right?

Reply:
You are absolutely right, resolved!